### PR TITLE
MRG: Pandas examples (continuation of #1556)

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
This is a continuation of #1556 to fix a reference in the docs.

### CODE REVIEW

**Comment by KamalakerDadi** [comment]
Can you please add into doc/whats_new.rst ?

File: examples/02_decoding/plot_haxby_full_analysis.py [hunk 1]
```diff
@@ -33,17 +33,17 @@
 
 # load labels
 import numpy as np
```
Comment (KamalakerDadi) [inline examples/02_decoding/plot_haxby_full_analysis.py]: Can you please import numpy at relevant section ?

File: examples/02_decoding/plot_haxby_searchlight.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 # Load Haxby dataset
 # -------------------
 import numpy as np
```
Comment (KamalakerDadi) [inline examples/02_decoding/plot_haxby_searchlight.py]: Can you please import numpy at relevant section/cell ? Makes it easier in user guide

File: examples/04_manipulating_images/plot_roi_extraction.py [hunk 3]
```diff
@@ -61,8 +61,9 @@
 
 # Second, load the labels stored in a text file into array using numpy
```
Comment (KamalakerDadi) [inline examples/04_manipulating_images/plot_roi_extraction.py]: using numpy -> using pandas. Also, can you please import numpy at relevant section ?

**Comment by KamalakerDadi** [comment]
Can you please also look into doc/manipulating_images/input_output.rst ? at section 5.1.3.3. Text files: phenotype or behavior

**Comment by KamalakerDadi** [comment]
Also, there are very few missed areas:
examples/02_decoding/plot_miyawaki_encoding.py
examples/02_decoding/plot_miyawaki_reconstruction.py

**Comment by KamalakerDadi** [comment]
I don't understand this circleci failure and I don't have precise reason to know why it is failing. I can't reproduce it locally either. 

May be few things, can fix the problem either using `align` to center or `scale` to 25%. Currently it is aligned to right & squeezed and I don't see it very clearly (because it is too small). 

If I remember correctly, I saw similar problem in Decoding PR which is not solved yet.

**Comment by KamalakerDadi** [comment]
Great!

**Comment by emdupre** [comment]
I've tried to address these comments in #1556, but I haven't updated either of the miyawaki examples:
examples/02_decoding/plot_miyawaki_encoding.py
examples/02_decoding/plot_miyawaki_reconstruction.py
the files that are being loaded here aren't typical use cases for pandas, so I thought it was better to handle them with numpy.

**Comment by GaelVaroquaux** [comment]
> Also, there are very few missed areas:
> examples/02_decoding/plot_miyawaki_encoding.py
> examples/02_decoding/plot_miyawaki_reconstruction.py

I think that we decided not to do these, as pandas didn't look useful for
them.

**Comment by KamalakerDadi** [comment]
LGTM

**Comment by GaelVaroquaux** [comment]
Closing, as superseeded.

**Pull request closed without merging**